aten/src/ATen/native_parse.py

```diff
@@ -54,11 +54,16 @@ def run(paths):
     for path in paths:
         for func in parse_native_yaml(path):
             declaration = {'mode': 'native'}
-            func_decl, return_type = [x.strip() for x in func['func'].split('->')]
+            if '->' in func['func']:
+                func_decl, return_type = [x.strip() for x in func['func'].split('->')]
+                return_type = sanitize_types(return_type)
```

Comment:
sanitize_types would be a good place to make everything `type: name:`. Also it would be easy to support named return types here as well.